### PR TITLE
security/vuxml: Minio vulnerability CVE-2022-24842

### PR DESCRIPTION
**Pull request by tomhukins**
PR:		268656
Reported by:	Adam Bishop

### CODE REVIEW

File: security/vuxml/vuln/2023.xml [hunk 1]
```diff
@@ -1,3 +1,30 @@
+  <vuln vid="8e20430d-a72b-11ed-a04f-40b034455553">
+    <topic>MinIO -- unprivileged users can create service accounts for admin users</topic>
+    <affects>
```
Comment (fernape) [inline security/vuxml/vuln/2023.xml:3]: `affects` tag is not ended

File: security/vuxml/vuln/2023.xml [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+      <package>
```
Comment (fernape) [inline security/vuxml/vuln/2023.xml:4]: `package` tag is not ended

File: security/vuxml/vuln/2023.xml [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+	<blockquote cite="https://github.com/minio/minio/security/advisories/GHSA-2j69-jjmg-534q">
+            <p>
+                A security issue was found where an unprivileged user is
+                able to create service accounts for root or other admin
+                users and then is able to assume their access policies
+                via the generated credentials.
+            </p>
+	</blockquote>
+      </body>
+    </description>
```
Comment (fernape) [inline security/vuxml/vuln/2023.xml:17]: `description` tag is not opened earlier

**Review by fernape — CHANGES_REQUESTED** [review]
Thanks for the entry. However there are (at least) a few things missing. Would you try and run `make validate` in `security/vuxml`?

**Comment by fernape** [comment]
Done in https://cgit.freebsd.org/ports/commit/?id=b16091e19db403fa19c514ec5ac4c15045e402ef

**Comment by tomhukins** [comment]
@fernape Thank you very much for fixing the XML and merging it.  I ran `make validate` on an earlier version of the patch, but I suspect I messed up resolving the conflicts when rebasing onto the updated main branch.

**Comment by fernape** [comment]
> @fernape Thank you very much for fixing the XML and merging it. I ran `make validate` on an earlier version of the patch, but I suspect I messed up resolving the conflicts when rebasing onto the updated main branch.

I believe you :-) Resolving conflicts in markup language files is super painful!

**Pull request closed without merging**